analytics.js

```diff
@@ -0,0 +1,1534 @@
+/* eslint-disable new-cap */
+/* eslint-disable func-names */
+/* eslint-disable eqeqeq */
+/* eslint-disable no-prototype-builtins */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable guard-for-in */
+/* eslint-disable no-sequences */
+/* eslint-disable no-multi-assign */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable import/extensions */
+/* eslint-disable no-param-reassign */
+import Emitter from "component-emitter";
+import querystring from "component-querystring";
```

Comment:
This file should not exists. Change should be made in src/analytics.js

Reply:
removed